store/src/java/com/zimbra/cs/util/BuildInfo.java

```diff
@@ -85,6 +87,11 @@
         }
     }
 
+    private static String getType() {
```

Comment:
Instead of hardcoded location. Can we use reflection and see if some class or java package of zimbra-license present in the classpath.

Reply:
That's a good idea!

Reply:
My suspicion is that will be no more reliable and might be more brittle if a caller doesn't have a classpath including the licensing extensions.  So, I would prefer not to do that unless there is a compelling reason for it.  I kind of think that any future change which involves a rename of the jar file will very quickly discover this code and change it appropriately.

Reply:
How about checking /opt/zimbra/bin/zmlicense utility? 
keeping flexibility on jar names or extension location.

Reply:
Yes, I think that would be better.  Will make that change.